test/langtools/tools/javac/processing/model/type/BasicAnnoTests.java

```diff
@@ -86,12 +89,30 @@ public class BasicAnnoTests extends JavacTestingAbstractProcessor {
     DPrinter dprinter;
     PrintWriter out;
     boolean verbose = true;
+    static Map<String, Class<? extends Annotation>> nameToAnnotation;
```

Comment:
this looks questionable ... a mutable static field in a commonly used super type

Reply:
Good point -- the mapping should be static given the method it is used in are static; I'll change to a static final. Thanks.

Reply:
is it possible to make this static final with an initializer?